src/entities/notesubfolder.cpp

```diff
@@ -112,6 +112,31 @@ QString NoteSubFolder::pathData() const {
                           : getParent().pathData() + QChar('\n') + _name;
 }
 
+/**
+ * Fetches a note sub folder by its id
+ */
+NoteSubFolder NoteSubFolder::fetchById(int id)
```

Comment:
In what way does this method differ from `NoteSubFolder::fetch`?

Reply:
@pbek Once again, I didn't see it :smile: